packages/css/src/toggle-group.css

```diff
@@ -26,18 +27,35 @@
   &:not([hidden]) {
     display: grid;
   }
+  &[data-variant='secondary'] {
+    --dsc-togglegroup-background: var(--ds-color-surface-tinted);
+    --dsc-togglegroup-border-color: var(--ds-color-border-subtle);
+    border: none;
+    outline: var(--dsc-togglegroup-border-width) var(--dsc-togglegroup-border-style) var(--dsc-togglegroup-border-color);
+    outline-offset: calc(var(--dsc-togglegroup-border-width) * -1);
+    button {
+      --dsc-button-color: var(--ds-color-text-default);
+      --dsc-button-color--hover: var(--ds-color-text-default);
+      border-radius: var(--dsc-togglegroup-border-radius);
+      min-height: calc(var(--dsc-togglegroup-button-size) + (var(--dsc-togglegroup-border-width) * 2));
+    }
+    button[aria-checked='true'] {
+      isolation: isolate;
```

Comment:
kult ☺️ gjerne skriv på kommentar ved bruk av avansert CSS så det blir lettere å lese og forså for de som skal bidra ❤️